cmd/lncli/types.go

```diff
@@ -38,3 +38,27 @@ func NewUtxoFromProto(utxo *lnrpc.Utxo) *Utxo {
 		Confirmations: utxo.Confirmations,
 	}
 }
+
+// PendingSweep displays information about a pending sweep within lnd's
+// UtxoSweeper.
+type PendingSweep struct {
+	OutPoint            OutPoint          `json:"outpoint"`
+	WitnessType         lnrpc.WitnessType `json:"witness_type"`
```

Comment:
If you change this to be a `string` and use the `String()` method on the enum, then the output will be `COMMITMENT_TIME_LOCK` rather than zero.

Reply:
True, though they currently already show the string value. Done anyway.